src/main/java/org/dependencytrack/notification/NotificationGroup.java

```diff
@@ -41,5 +41,6 @@ public enum NotificationGroup {
     BOM_PROCESSED,
     VEX_CONSUMED,
     VEX_PROCESSED,
-    POLICY_VIOLATION
+    POLICY_VIOLATION,
+    NEW_PROJECT
```

Comment:
IMO the name should be `PROJECT_CREATED` instead. It is more idiomatic and allows for further additions like `PROJECT_DELETED` or similar changes affecting projects in the future.